testing/xfpga/test_error_c.cpp

```diff
@@ -87,6 +87,7 @@ class error_c_mock_p
       filter_ = nullptr;
     }
     token_cleanup();
+    tmpsysfs_ = "";
```

Comment:
Interesting.. the error goes away with this change?

Reply:
Yes. Apparently it doesn't think it gets cleaned up unless we do this.

Reply:
Interesting.  Well, these test fixtures are in global scope, so I suppose that's what the checker is complaining about.